packages/amplication-server/src/core/resource/resource.service.ts

```diff
@@ -539,16 +539,35 @@ export class ResourceService {
       event: EnumEventType.ArchitectureRedesignApply,
     });
 
+    const originalResourceId = movedEntities[0]?.originalResourceId;
```

Comment:
Should we use `resourceId` from line 526 here?

Reply:
@mulygottlieb no, because it gives us a default value if there is no new entities, we don't want this in this case